sdk/python/tests/integration/registration/test_universal_registry.py

```diff
@@ -1767,3 +1767,92 @@ def test_apply_entity_success_with_purge_feast_metadata(test_registry):
     assert len(entities) == 0
 
     test_registry.teardown()
+
+
+combined_sql_fixtures = [
```

Comment:
Can we concatenate three existing lists of fixtures here instead of redefining?

Reply:
Done. Thank you.